### PR TITLE
Update graphviz info and provide IP

### PR DESCRIPTION
**Pull request by martindurant**

### CODE REVIEW

**Comment by martindurant** [comment]
@TomAugspurger @jcrist , does this sound right?

**Comment by TomAugspurger** [comment]
Yep, I think that looks good.

File: README.md [hunk 1]
```diff
@@ -62,8 +67,25 @@ python prep_data.py
...
 
+### Installing Graphviz
+
+Graphviz is used by Dask to produce graphical representations graphs in the notebook. It is an optional extra that
+you can install.
+
+Although graphviz and it's python bindings are included in the provided environment, you need extra libraries for it
+to work on your system, and what you need depends on your OS
+
+- for linux, `conda install libiconv pango -c conda forge` should be enough, *or* you can get graphviz from your
```
Comment (jcrist) [inline README.md]: `conda forge` -> `conda-forge`

File: README.md [hunk 2]
```diff
@@ -62,8 +67,25 @@ python prep_data.py
...
+system package manager
+- for OSX, `conda install libiconv pango -c conda forge` should also work, *or* you can install graphviz with
```
Comment (jcrist) [inline README.md]: Same here

**Pull request merged**